libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/offroute/OffRouteDetector.java

```diff
@@ -226,24 +224,28 @@ private static boolean movingAwayFromManeuver(RouteProgress routeProgress,
     LineString remainingStepLineString = TurfMisc.lineSlice(userPointOnStep, maneuverPoint, stepLineString);
     double userDistanceToManeuver = TurfMeasurement.length(remainingStepLineString, TurfConstants.UNIT_METERS);
```

Comment:
Since we are constantly casting this to int anyway, should we just store it here?